src/wallet/api/walletServiceAxios.ts

```diff
@@ -46,4 +51,17 @@ export const axiosInstance = async (wallet: HathorWalletServiceWallet | null = n
   return axios.create(defaultOptions);
 }
 
-export default axiosInstance;
\ No newline at end of file
+/**
+ * Returns the correct base url constant for wallet service based on the network
+ *
+ * @param {Network} network The network, can be either mainnet or testnet but will default to testnet
```

Comment:
There is no default, the parameter is required, right?

Reply:
Yes.

What do you think about it?

Reply:
I like it.